runtime/src/tests/work-item/work-item-list/workitem-naughty-string.spec.js

```diff
@@ -1,102 +0,0 @@
-/**
- * POC test for automated UI tests for Planner
- *  Story: Display and Update Work Item Details
- *  https://github.com/almighty/almighty-core/issues/296
- *
- * Note on screen resolutions - See: http://www.itunesextractor.com/iphone-ipad-resolution.html
- * Tests will be run on these resolutions:
- * - iPhone6s - 375x667
- * - iPad air - 768x1024
- * - Desktop -  1920x1080
- *
- * beforeEach will set the mode to phone. Any tests requiring a different resolution will must set explicitly.
- *
- * @author
- */
-
-var WorkItemListPage = require('./page-objects/work-item-list.page'),
-  testSupport = require('./testSupport');
-
-describe('Work item list', function () {
-  var page, items, browserMode;
-  var until = protractor.ExpectedConditions;
-  var waitTime = 30000;
-
-/* Troublesome strings reference: https://github.com/minimaxir/big-list-of-naughty-strings */
-var UnicodeSymbols = "Ω≈ç√∫˜µ≤≥÷åß∂ƒ©˙∆˚¬…æœ∑´®†¥¨ˆøπ“‘¡™£¢∞§¶•ªº–≠¸˛Ç◊ı˜Â¯˘¿ÅÍÎÏ˝ÓÔÒÚÆ☃Œ„´‰ˇÁ¨ˆØ∏”’`⁄€‹›ﬁﬂ‡°·‚—±⅛⅜⅝⅞ЁЂЃЄЅІЇЈЉЊЋЌЍЎЏАБВГДЕЖЗИЙКЛМНОПРСТУФХЦЧШЩЪЫЬЭЮЯабвгдежзийклмнопрстуфхцчшщъыьэюя٠١٢٣٤٥٦٧٨٩";
-var TwoByteCharacters = "田中さんにあげて下さいパーティーへ行かないか和製漢語部落格사회과학원 어학연구소찦차를 타고 온 펲시맨과 쑛다리 똠방각하社會科學院語學研";
-var JapaneseEmoticons = "ヽ༼ຈل͜ຈ༽ﾉ ヽ༼ຈل͜ຈ༽ﾉ(｡◕ ∀ ◕｡)｀ｨ(´∀｀∩__ﾛ(,_,*)・(￣∀￣)・:*:ﾟ･✿ヾ╲(｡◕‿◕｡)╱✿･ﾟ,。・:*:・゜’( ☻ ω ☻ )。・:*:・゜’(╯°□°）╯︵ ┻━┻)(ﾉಥ益ಥ）ﾉ﻿ ┻━┻┬─┬ノ( º _ ºノ)( ͡° ͜ʖ ͡°)";
```

Comment:
@Raunak1203 

Have you ported the naughty strings tests to the new TS tests?

Reply:
@rgarg1 here is the PR https://github.com/fabric8-ui/fabric8-planner/pull/2733

Reply:
Excellent